control-plane/roles/alerting/defaults/main/main.yaml

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
I thought there was no alerting role? Also all these variables are not used.

Reply:
https://github.com/metal-stack/metal-roles/commit/f35e01dfab8cdadaa0ae5e24c3ef331e25243322